### DISCUSSION TITLE
Outlook - SupportsNoItemContext - shared mailbox email address

### DISCUSSION THREAD

**Discussion by PiotrX1** [OfficeDev/office-js#3727] [Q&A]
Is there any way to check which shared mailbox message came from? I'm using getSelectedItemsAsync to get mails. SelectedItemDetails does not include this.

**Comment by alex-fitzgerald** [comment]
I think you can make an [EWS request to get the details of the email](https://learn.microsoft.com/en-us/office/dev/add-ins/outlook/web-services#using-the-makeewsrequestasync-method-to-access-ews-operations), which will include the mailbox the email was received in.
